src/NextGen/DonationForm/Repositories/DonationFormRepository.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Give\NextGen\DonationForm\Repositories;
+
+use Give\Framework\FieldsAPI\Amount;
+use Give\Framework\FieldsAPI\Contracts\Node;
+use Give\Framework\FieldsAPI\DonationSummary;
+use Give\Framework\FieldsAPI\Email;
+use Give\Framework\FieldsAPI\Exceptions\EmptyNameException;
+use Give\Framework\FieldsAPI\Exceptions\TypeNotSupported;
+use Give\Framework\FieldsAPI\Form;
+use Give\Framework\FieldsAPI\Hidden;
+use Give\Framework\FieldsAPI\Name;
+use Give\Framework\FieldsAPI\Paragraph;
+use Give\Framework\FieldsAPI\PaymentGateways;
+use Give\Framework\FieldsAPI\Section;
+use Give\Framework\FieldsAPI\Text;
+use Give\Framework\PaymentGateways\PaymentGateway;
+use Give\Framework\PaymentGateways\PaymentGatewayRegister;
+use stdClass;
+
+class DonationFormRepository {
```

Comment:
Hmm, I know @kjohnson is building out the Model. I suspect a lot of this will be useful there. Let's make sure we avoid weird conflicts.

Reply:
Yeah I just needed to dump all our logic that was inside the block render somewhere and a repository made the most sense.  I figured kyle will be using this for the form model & updating anyway.